contrib/completion/bash/docker

```diff
@@ -4202,6 +4203,13 @@ _docker_manifest_push() {
 	esac
 }
 
+_docker_manifest_rm() {
```

Comment:
Completion should handle `--help` here, not just a manifest. Please surround with a `case` block like in the other comands but with just `--help` .